features/step_definitions/basic_steps.rb

```diff
@@ -2,6 +2,10 @@
   visit root_path
 end
 
+When("I click {string} link") do |link|
+  click_link link
+end
+
 Given("I visit the Become a host page") do
   visit new_listing_path
```

Comment:
But new listing is a path for make a new listing? When you click Become a host you have to create host profile, right? When you click on New listing, you get a form to fill in for a new cat listing?

Reply:
Changed, both in navbar and tests.